scripts/basic-workflow-tests.sh

```diff
@@ -343,6 +343,8 @@ wait_for_server(){
 
 usage() {
     echo "Usage: $0 [-m] [-h]"
+    echo "  NOTE: If you add additional or modify existing options, please document them in 'docs/ci.md'"
```

Comment:
I think a code comment is fine. Having this in the help output seems a little odd

Reply:
Personally I like having it here, as it will be mostly devs running this. But that being said, we can move it if you feel it would be best @russellb .